pnpm-lock.yaml

```diff
@@ -10355,17 +10426,17 @@ packages:
       - supports-color
     dev: true
 
-  /@typescript-eslint/typescript-estree@5.60.1(typescript@5.1.3):
-    resolution: {integrity: sha512-hkX70J9+2M2ZT6fhti5Q2FoU9zb+GeZK2SLP1WZlvUDqdMbEKhexZODD1WodNRyO8eS+4nScvT0dts8IdaBzfw==}
+  /@typescript-eslint/typescript-estree@5.62.0(typescript@5.1.3):
```

Comment:
this could be causing problems

Reply:
This is normal.

We have dev dependencies to some packages that depend on the older `@typescript-eslint/*@v5.x.x` but that doesn't influence the main hoisted version.

This is not related to `pnpm` only. All the package managers keep hierarchical information to ensure version clashes are avoided.